configure.ac

```diff
@@ -82,20 +82,20 @@ case $system in
     ;;
 esac
 
-CCFLAGS="$CCFLAGS $OPENMP_CFLAGS"
-
 if test $ENABLE_DEBUG = yes; then
-  CCFLAGS="$CCFLAGS -g"
+  DEBUG_CFLAGS="-g"
 fi
 
 if test $ENABLE_PIC = yes; then
-  CCFLAGS="$CCFLAGS -fPIC"
+  PIC_CFLAGS="-fPIC"
```

Comment:
Are you *sure* this works? Did you do make clean? I would think this removes the `-fPIC` option when building the main libsharp library, which should make it unlinkable, at least on most platforms.

Since not passing -fPIC is a bug, I'm not sure why it shouldn't be in `CCFLAGS`.